driver-examples/tycho/driver.target

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Using a pom-first approach in Tycho for some dependencies makes it harder for Eclipse to correctly guess the classpath. My workaround consists of two pieces:
- A special "repository" submodule that creates a local P2 installation with the driver and its dependencies
- This target file that references the P2 installation
It works, but users are required to
- Generate the P2 directory with Tycho
- Set this platform definition as the current active target platform definition